### PR TITLE
GEN-768 - feat: redirect home routes

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2793** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2793" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2793?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/next.config.js [hunk 1]
```diff
@@ -124,6 +124,42 @@ module.exports = withBundleAnalyzer({
...
+              destination: '/se/forsakringar/hemforsakring',
+              permanent: false,
+              locale: false,
+            },
+            {
+              source: '/se-en/new-member/home-insurance',
+              destination: '/se-en/insurances/home-insurance',
+              permanent: false,
+              locale: false,
+            },
```
Comment (robinandeer) [inline apps/store/next.config.js:162]: Are we sure we don't pass code-query param for these routes?

**Review by robinandeer — COMMENTED** [review]
Shouldn't this be a stack?

**Comment by guilhermespopolin** [comment]
Replace by [this](https://github.com/HedvigInsurance/racoon/pull/2796).
Sorry for the trouble @alebedev @robinandeer

**Pull request closed without merging**